Cargo.toml

```diff
@@ -24,3 +24,6 @@ colored = "2.0.0"
 [dev-dependencies]
 anyhow = "1.0.36"
 pretty_assertions = "0.6.1"
+
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(tarpaulin_include)'] }
```

Comment:
Not an objection, but what is your justification for adding this lint rule?

Reply:
I wasn't sure what `tarpaulin_include` was for, and when I grep'd the repository I couldn't find it. Do you think it should just be removed? It was also issuing lint warnings, but I wasn't aware of its context to remove it.